src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1029,8 +1030,8 @@ def _validate_and_get_connection_string(resource_group_name, storage_account):
         raise CLIError(error_message)
 
     keys = storage_client.storage_accounts.list_keys(resource_group_name, storage_account).keys
```

Comment:
Slight change to wording:
> Storage account has no '{}' endpoint. A storage account that supports blob, queue, and table storage is required.

Even better:
Storage account 'name' has no '{}' endpoint...